privaterelay/templates/includes/alias-card.html

```diff
@@ -135,25 +137,25 @@
           </div>
           <div class="c-alias-block-settings-inputs">
             <div class="c-alias-block-settings-input t-line-left t-forward">
-              <input type="radio" name="aliasBlockSettings{{ alias.id }}" id="forwardAll{{ alias.id }}" value="forwardAll" checked />
+              <input type="radio" name="aliasBlockSettings{{ alias.id }}" id="forwardAll{{ alias.id }}" value="forwardAll" {% if alias.enabled %} {% if not alias.block_list_emails %}checked{% endif %} {% endif %} />
```

Comment:
Heh, totally a nit, but I think you can do conjunctions in Django templates :P 

```suggestion
              <input type="radio" name="aliasBlockSettings{{ alias.id }}" id="forwardAll{{ alias.id }}" value="forwardAll" {% if alias.enabled and not alias.block_list_emails %}checked{% endif %} />
```

Reply:
https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#boolean-operators